packages/bpk-component-accordion/src/BpkAccordionItem.module.scss

```diff
@@ -68,9 +78,25 @@ $bpk-spacing-v2: true;
     &--flipped {
       transform: scaleY(-1);
     }
+
+    &--day {
+      fill: $bpk-canvas-day;
+    }
   }
 
   &__content-container {
     margin: 0;
+
+    &--expanded {
+      padding-bottom: $bpk-spacing-base-v-2;
+
+      @include bpk-border-bottom-sm($bpk-line-day);
+
+      &-day {
+        padding-bottom: $bpk-spacing-base-v-2;
+
+        @include bpk-border-bottom-sm($bpk-line-on-dark-night);
```

Comment:
Night tokens are not to be used outside of dark mode, can you check the designs of what should be used here and if it's a night token we can raise to design

Reply:
Ah I may have just looked at RBG values. I'll have a better look 👀

Reply:
Found what I was doing wrong 🤦